src/intervals/rounding.jl

```diff
@@ -157,23 +199,31 @@ doc"""
     setrounding(Interval, rounding_type::Symbol)
 
 Set the rounding type used for all interval calculations on Julia v0.6 and above.
-Valid `rounding_type`s are `:correct`, `:fast` and `:none`.
+Valid `rounding_type`s are `:correct`, `:fast` and `:none`, `:errorfree`.
```

Comment:
I think this should read as "Valid `rounding_type`s are `:correct`, `:fast`, `:none`, and `:errorfree`".